blog_post_collaboration/kedro_ibis.md

```diff
@@ -0,0 +1,263 @@
+One](https://www.databricks.com/blog/2019/08/22/guest-blog-how-virgin-hyperloop-one-reduced-processing-time-from-hours-to-minutes-with-koalas.html), demonstrate how these solutions can accelerate the scaling process for data-intensive code. That said, building and maintaining a pandas-compatible API requires significant commitment from the engine developer, so no panacea exists.
+
+## The SQL solution
+
+Thankfully, there exists a standardized programming language that every database (and more-or-less every major computation framework) supports: SQL![^1]
+
+_If you like SQL,_ [dbt](https://www.getdbt.com) provides a battle-tested framework for defining and deploying data transformation workflows. [As of February 2022, over 9,000 companies were using dbt in production.](https://www.getdbt.com/blog/next-layer-of-the-modern-data-stack) Here on the Kedro team, we love dbt! In fact, we frequently recommend it for ELT (Extract, Load, Transform) workflows. Kedro and dbt share similar self-stated goals: [Kedro empowers data teams to write reproducible, maintainable, and modular Python code,](https://docs.kedro.org/en/stable/introduction/index.html#introduction-to-kedro) and ["dbt is a SQL-first transformation workflow that lets teams quickly and collaboratively deploy analytics code following software engineering best practices."](https://www.getdbt.com/product/what-is-dbt)
+
+> _"When I learned about Kedro (while at dbt Labs), I commented that it was like dbt if it were created by Python data scientists instead of SQL data analysts (including both being created out of consulting companies)."_
+>
+> —Cody Peterson, Technical Product Manager
+
+## "What if I don't want to use SQL?"
+
+This post isn't going to enter the SQL vs. Python debate. Nevertheless, many teams and data scientists choose Python for their analytics workflows. Furthermore, in my experience, teams using Kedro for their machine learning pipelines prefer to use the same tool for data processing and feature engineering code, too.
+
+Can we combine the flexibility and familiarity of Python with the scale and performance of modern SQL? Yes we can! ["Ibis is a Python library that provides a lightweight, universal interface for data wrangling."](https://github.com/ibis-project/ibis) Ibis exposes a Python dataframe API where the same code can execute against over 15 query engines, from local backends like pandas, Polars, and DuckDB to remote databases and distributed computation frameworks like BigQuery, Snowflake, and Spark. Ibis constructs a query plan, or intermediate representation (IR), that it evaluates lazily (i.e. as and when needed) on the execution engine.
```

Comment:
might be worth mentioning that Ibis also makes it easy to use SQL strings in the same codebase with `.sql` and `.to_sql` methods -- i.e. you're not losing the ability to leverage SQL